lib/internal/util.js

```diff
@@ -201,17 +201,19 @@ function getConstructorOf(obj) {
 const kCustomPromisifiedSymbol = Symbol('util.promisify.custom');
 const kCustomPromisifyArgsSymbol = Symbol('customPromisifyArgs');
 
-function promisify(orig) {
-  if (typeof orig !== 'function') {
+function promisify(original) {
```

Comment:
this change looks entirely unrelated to the internal/errors change... it should likely be in it's own PR